src/main/java/com/codecool/expertsystem/view/Display.java

```diff
@@ -8,17 +8,28 @@
     private static String intro = "Welcome to EXPERT CAR SYSTEM\nby Fast And Furious";
 
     public static void printMessage(String message) {
+
         System.out.println(message);
+
     }
 
     public static void printTitleMenu() {
+
         System.out.println(intro);
         System.out.println(verticalBarLine);
+
     }
 
     public static String getStringInput(String message) {
+
         Scanner userInput = new Scanner(System.in);
         System.out.println(message);
-        return userInput.nextLine();
+        return userInput.nextLine().toLowerCase();
+
     }
+
+    public static void clearScreen() {
+        System.out.print("\033[H\033[2J");
+        System.out.flush();
```

Comment:
flush() is unecessary

Reply:
it is not necessary in most (if not all) linux systems, but i knew there is probability in Windows.

Reply:
System.out is based around a PrintStream which by default flushes whenever a newline is written.

From the javadoc:

    autoFlush - A boolean; if true, the output buffer will be flushed whenever a byte array is written, one of the println methods is invoked, or a newline character or byte ('\n') is written

So the println case you mention is explicitly handled, and the write case with a byte[] is also guaranteed to flush because it falls under "whenever a byte array is written".

If you replace System.out using System.setOut and don't use an autoflushing stream, then you will have to flush it like any other stream.

Library code probably shouldn't be using System.out directly, but if it does, then it should be careful to flush because a library user might override System.out to use a non flushing stream.

Any Java program that writes binary output to System.out should be careful to flush before exit because binary output often does not include a trailing newline.

Reply:
in Windows i use bash emulator. It is salty about it :)